shared/utilities/getTicketStatus/test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint max-len: ["error", 2000] */
+import { getTicketStatusOptions, getActionLink } from '.';
+import tk from 'timekeeper';
+
+describe('getTicketStatusOptions', () => {
+  it('returns Free Ticket and empty linkType if the current time is before the ticket release time', () => {
+    tk.freeze(new Date(100));
+    const ticketReleaseDate = new Date(101);
+    const result = getTicketStatusOptions({ ticketReleaseDate });
+
+    expect(result).to.deep.equal({
+      buttonText: 'Free Ticket',
+      linkType: '',
+      statusHeader: 'TICKETS WILL GO LIVE ON',
+      statusSubHeader: 'Thursday, 1st January 1970, 01:00',
+      link: undefined,
+    });
+  });
+
+  it('returns Free ticket and Event linkType if the current Time is greater than the release date and there are tickets availiable', () => {
```

Comment:
Break long descriptions onto multiple lines for legibility and to conform to the style guide